README.md

```diff
@@ -87,6 +87,13 @@ make tilt-up
 
 Notice that local images are build and pushed to docker running inside the minikube cluster since we are running `eval $(minikube docker-env)` before launching Tilt.
 
+### Dashboard
```

Comment:
maybe moving this to the Usage section? I think this part of the documentation (develop) is more for a developer than a user.

Reply:
Done in d4680b6998d7e604f03826f956124af7a177e0d9.